doc/python/migrate-to-maplibre.md

```diff
@@ -0,0 +1,71 @@
+    description: Migrating from Mapbox traces to MapLibre traces.
+    display_as: basic
+    language: python
+    layout: base
+    name: MapLibre Migration
+    order: 1
+    page_type: example_index
+    permalink: python/mapbox-to-maplibre/
+    redirect_from: python/mapbox-migration/
+    thumbnail: thumbnail/line-and-scatter.jpg
+---
+
+## Migrating from Mapbox traces to MapLibre traces
+
+With the release of Plotly.py v5.24.0, we are introducing a new set of trace types for maps with tile underlays, including from Plotly Express:
+- `px.scatter_map`
+- `px.line_map`
+- `px.choropleth_map`
+- `px.density_map`
+
+as well as Plotly Graph Objects:
+- `go.Choroplethmap`
+- `go.Scattermap`
+- `go.Densitymap`
+
+These traces replace the existing Mapbox traces, `px.scatter_mapbox`, `px.line_mapbox`, etc., but use [MapLibre](https://maplibre.org) as the map renderer rather than Mapbox.
+
+When switching to the new traces, keep an eye out for improved rendering performance, WebGL2 support, and over time, improved features in the Plotly map traces inherited from the MapLibre renderer, including projection support, globe views, terrain support, and support for modern mapping standards.
+
+You can learn  more about the motivations for this change in our [announcement post](https://plotly.com/blog/plotly-is-switching-to-maplibre/).
+
+As a result of removing Mapbox as the rendering engine, we're also removing the Mapbox branding from these trace names. This means that migrating from Mapbox traces to MapLibre traces will require some code changes in your projects.
+
+1. Change trace names from `*mapbox` to `*map`. For any existing trace name ending in `*mapbox`, ensure you've removed the "`box`" suffix.
+2. If in use, update `layout.mapbox` argument in your layout configuration to `layout.map`. The nested properties are identical in the new map traces, so no other changes should be required.
+3. If in use, update `mapbox_style` to `map_style`.
+4. Verify your `map_style` settings. With `mapbox` traces, we bundle `basic`, `streets`, `outdoors`, `light`, `dark`, `satellite`, and `satellite-streets` styles, using Mapbox styling. These style names are still available, but they now reference slightly different styles provided by other tools.
+
+Note that Mapbox API keys are no longer required for Plotly-provided styles, but using external styles in your Plotly maps, remains supported with the existing API.
+
+### Style changes
+Built-in styles in map traces are free styles from [Carto](https://carto.com) and [ESRI](https://www.esri.com/en-us/home). Several names are re-used from the previous Mapbox styles.
+<p align="center">
+  <img src="https://raw.githubusercontent.com/plotly/plotly.js/master/test/image/baselines/map_predefined-styles1.png" alt="Style comparison part 1" width="45%" />
+  <img src="https://raw.githubusercontent.com/plotly/plotly.js/master/test/image/baselines/map_predefined-styles2.png" alt="Style comparison part 2" width="45%" />
+</p>
```

Comment:
I would like to verify that these styles look okay in a preview deploy. I did not test locally.

Reply:
I don't like using direct links to master. 
Instead you may use either a release tag or a commit hash so that in future if `mapbox` images are removed it would not impact this.

Reply:
✅ Good call

Reply:
Also see https://github.com/plotly/plotly.js/commit/c31bc9fac5dafd75ae525208f198c1adbf8c4c22 for styles in map subplots vs styles in mapbox subplots.

Reply:
I'd prefer even to copy these images and host elsewhere as they're test artifacts.

Reply:
@LiamConnors what's the best place to host these images?

Reply:
Probably best place is add them here: https://github.com/plotly/graphing-library-docs/tree/master/all_static/images and then reference those in the same way as you're currently doing with the new links